position_controller:
  type: om_position_controller/PositionController
  joint_names:
    - joint1
    - joint2
    - joint3
    - joint4
    - joint5
    - joint6
    - gripper
    
  root_link: world
  tip_link: end_effector_link